test/validate/eia_test.py

```diff
@@ -45,17 +45,17 @@ def test_no_null_cols_eia(pudl_out_eia, live_dbs, cols, df_name):
 @pytest.mark.parametrize(
     "df_name,raw_rows,monthly_rows,annual_rows",
     [
-        ("bf_eia923", 1_569_568, 1_569_568, 128_252),
+        ("bf_eia923", 1_607_752, 1_607_752, 128_252),
```

Comment:
i went and quickly looked at [the build logs](https://storage.cloud.google.com/builds.catalyst.coop/2024-02-09-1403-efb1a075a-3381-no-coverage-in-nightlies/2024-02-09-1403-efb1a075a-3381-no-coverage-in-nightlies-pudl-etl.log) and it looks like this one is a typo and should be `1_670_752`

```python
E           ValueError: bf_eia923: found 1607752 rows, expected 1569568. Off by 2.433%, allowed margin of 0.000%
```

Reply:
In the example above the found value `1607752` is the same as the one I have written, it's just confusing to look at with the underscores!

Reply:
oh you are so right! sorry!